config/environments/production.rb

```diff
@@ -60,6 +60,19 @@
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
 
+  config.action_mailer.default_url_options = {host: 'datarockets-finance.herokuapp.com', port: 3000}
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: 'smtp.gmail.com',
+    port: 587,
```

Comment:
@sinpai port numbers are usually kept in ENV variables. Check comment from the previous review about `3000` port